### PR TITLE
Update make lint to use tox

### PR DESCRIPTION
**Pull request by charles-cooper**
### What I did
`make lint` is no longer the same as CI. This fixes that :)

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-bumpversion
```
Comment (davesque) [inline requirements-dev.txt]: I think `requirements-dev.txt` should be deleted actually.  Hasn't it been supplanted with `pip install -e .[test,lint]`?

Comment (davesque) [reply requirements-dev.txt]: And where's your cute animal??

Comment (charles-cooper) [reply requirements-dev.txt]: Neat, thanks. Done in c12534c5e9

Comment (charles-cooper) [reply requirements-dev.txt]: I was in a rush and left my cute animal at home! Added

**Pull request merged**